src/Util/ClassSourceManipulator.php

```diff
@@ -130,29 +122,15 @@ public function addEmbeddedEntity(string $propertyName, string $className): void
     {
         $typeHint = $this->addUseStatementIfNecessary($className);
 
-        $annotations = [];
-        $attributes = [];
-
-        if (!$this->useAttributesForDoctrineMapping) {
-            $annotations = [
-                $this->buildAnnotationLine(
-                    '@ORM\\Embedded',
-                    [
-                        'class' => new ClassNameValue($className, $typeHint),
-                    ]
-                ),
-            ];
-        } else {
-            $attributes = [
+        $attributes = [
                 $this->buildAttributeNode(
                     Embedded::class,
                     ['class' => new ClassNameValue($className, $typeHint)],
                     'ORM'
                 ),
             ];
```

Comment:
I think indentation is weird here too now?

Reply:
done